open-learning-exchange/myplanet
# all: persistent dark mode (fixes #4133)

fixes #4133 
how to test 
1. change themes in different languages
2. change to dark mode then close app and reopen to see if the darkmode persists
